docs/index.md

```diff
@@ -1275,6 +1275,18 @@ $ mocha --reporter list --growl
 
 By default, `mocha` looks for the glob `./test/*.js`, so you may want to put your tests in `test/` folder. If you want to include sub directories, use `--recursive`, since `./test/*.js` only matches files in the first level of `test` and `./test/**/*.js` only matches files in the second level of `test`.
 
+To configure where `mocha` looks for tests, you may pass your own glob:
+
+```
+$ mocha ./spec/*.js
+```
+
+If you use shell expansion for this, you must wrap your glob pattern in double quotes (note, using the `--recursive` flag here is useless):
```

Comment:
A lot of people probably doesn't know what "shell expansion" is.

In fact, I would probably remove L1279-1283 and just give the example below with double quotes.  Then, explain below the example how using `**` means `--recursive` is useless.  Recommend using double quotes for "portability"; otherwise it'll need a pretty long-winded explanation.